install-cloud/test/main.test.ts

```diff
@@ -151,7 +169,8 @@ test('cloud API url retrieved and assigned to output', async () => {
 test('RDS credentials retrieved and assigned to output', async () => {
   await main()
 
-  expect(execSync).toHaveBeenCalledWith(
+  // throw error when missing flag "--skip-logs"
+  expect(execSync).not.toHaveBeenCalledWith(
```

Comment:
Why not test for `toHaveBeenCalledWith` with --skip-logs? The test looks misleading